mappings/net/minecraft/block/Block.mapping

```diff
@@ -30,14 +41,21 @@ CLASS none/akw net/minecraft/block/Block
 	METHOD b (Lnone/asj;Lnone/aiu;Lnone/cn;)Lnone/bcp;
 		ARG 0 state
 		ARG 1 world
+	METHOD b getBoundingBox (Lnone/asj;Lnone/aiy;Lnone/cn;)Lnone/bcp;
+	METHOD c getLocalizedName ()Ljava/lang/String;
```

Comment:
Either works, I just went with unlocalized and localized because that's what they are and those are the de facto standard.